dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -684,6 +715,28 @@ public ResponseEntity<RepresentationModel<?>> patch(HttpServletRequest request,
         return patchInternal(request, apiCategory, model, id, jsonNode);
     }
 
+    /**
+     * PATCH method, using operation on the resources following (JSON) Patch notation (https://tools.ietf
+     * .org/html/rfc6902)
+     *
+     * Note that the regular expression in the request mapping accept a UUID as identifier;
+     *
+     * @param request
+     * @param apiCategory
+     * @param model
+     * @param id
+     * @param jsonNode
+     * @return
+     * @throws HttpRequestMethodNotSupportedException
+     */
+    @RequestMapping(method = RequestMethod.PATCH, value = REGEX_REQUESTMAPPING_IDENTIFIER_AS_STRING_VERSION_STRONG)
+    public ResponseEntity<RepresentationModel<?>> patch(HttpServletRequest request, @PathVariable String apiCategory,
```

Comment:
Same here. If possible, we should add tests for this new method to `RestResourceControllerIT`

Reply:
methods `RestResourceController.delete(..., String);` | `RestResourceController.upload(..., String);` | `RestResourceController.patch(..., String);` have been added because they are used into `QAEventRestRepository`. Goal is the reflection is able to manage these calls and handle them properly to `QAEventRestRepository`. Consider **`QAEventRestRepositoryIT`** tests these methods already.